tests/sqlparser_postgres.rs

```diff
@@ -3773,3 +3773,14 @@ fn test_simple_insert_with_quoted_alias() {
         }
     )
 }
+
+#[test]
+fn parse_array_agg() {
+    // follows general function with wildcard code path
+    let sql = r#"SELECT GREATEST(sections_tbl.*) AS sections FROM sections_tbl"#;
+    pg().verified_stmt(sql);
+
+    // follows special-case array_agg code path
```

Comment:
Could you please also add a test with a multi-part identifier? LIke

```sql
SELECT GREATEST(my_schema.sections_tbl.*) AS sections FROM sections_tbl
```

Reply:
Good idea, I added such tests for both the general path and the `array_agg` path.